view/perspective/utils.js

```diff
@@ -77,9 +77,7 @@ function getArray(field, arrayOfObjects) {
   }
 
   for (let i = 0; i < arrayOfObjects.length; i++) {
-    if (arrayOfObjects[i].isPublished) {
-      arr.push(arrayOfObjects[i][field]);
-    }
+    arr.push(arrayOfObjects[i][field]);
```

Comment:
isPublished check has been moved to getValuesObject() in view/perspective/utils.js